src/librustc/dep_graph/graph.rs

```diff
@@ -600,7 +600,7 @@ impl DepGraph {
                             DepKind::Hir |
                             DepKind::HirBody |
                             DepKind::CrateMetadata => {
-                                if dep_node.extract_def_id(tcx).is_none() {
+                                if dep_dep_node.extract_def_id(tcx).is_none() {
```

Comment:
One could argue, that the parameter name is not very well choosen :/ Maybe one should address as well in a next patch

Reply:
It might make sense to rename `dep_dep_node` to `dep_node_of_dependency` or something like that, yeah.